features.html

```diff
@@ -9,11 +9,15 @@
 
 
 <div class="feature-group">
+  <div id="smart-editor" class="feature-anchor"></div>
+  <div class="feature-link"><a class="icon-link" href="#smart-editor"></a></div>
```

Comment:
This is a bit repetitive, isn't it?
What do you think about introducing new partial, like already existing `h` one?